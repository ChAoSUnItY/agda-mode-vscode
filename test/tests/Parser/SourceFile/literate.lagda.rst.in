This line is ordinary text, which is ignored by Agda.
?

::

  module Whatever where
  -- Agda code goes here

  ?

Another non-code line.?
::
.. This line is also ignored ?

..
  ::
  -- Agda code

  ?


The second one is the language containing a single word; the empty word. The
root node is labelled ``true``, and all the others are labelled ``false``::

    ε : ∀ {i A} → Lang i A
    ν ε   = ?
    δ ε _ = ∅