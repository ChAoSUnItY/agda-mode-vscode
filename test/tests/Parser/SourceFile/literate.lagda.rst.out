Hole [0] (113, 114) => (113, 120) "{!   !}"
Hole [1] (207, 208) => (207, 214) "{!   !}"
Hole [2] (410, 411) => (410, 417) "{!   !}"